packages/button/README.md

```diff
@@ -26,6 +26,8 @@ Each page will likely have one primary button. Any remaining calls-to-action are
 
 **Processing**: When user clicks or taps a button and an API request is initiated we should provide feedback so they know their action was received.
```

Comment:
We need this when action will take longer than two seconds to complete and text should adjust accordingly and be disabled
Saving... 
Submitting...
Deleting...

Reply:
We can't know for sure whether an action will take > 2 seconds

Reply:
Well I guess it depends on how implementing the processing state. 

Is it opt in by engineers or processing state always occurs?

If it's opt-in, typically we have a good understanding of an expected amount of time so we can make an educated guess. if we can't, we have deeper problems. 
https://uxmovement.com/buttons/when-you-need-to-show-a-buttons-loading-state/

If processing state should always occur as you mention, upon API call, sounds good there but we should give examples like "Saving..."  Ideally at the component level, devs get this for free with our standard actions. e.g, if standards save button in processing state, button will say "Saving..." and cursor will show as busy if hover over. 

Do other page controls become disabled too for free or need to define that? good tuff to expand on in guidelines.

I don't think we need to delay publishing of guideline but good section to add into and come up with our diff cases and considerations.

https://material.io/components/progress-indicators#usage